scm-core/src/main/java/sonia/scm/ConcurrentModificationException.java

```diff
@@ -33,7 +33,7 @@
 
   private static final String CODE = "2wR7UzpPG1";
 
-  public ConcurrentModificationException(Class type, String id) {
+  public ConcurrentModificationException(Class<?> type, String id) {
```

Comment:
Could you explain this change to me?

Reply:
The use of a generic class without generics creates a warning. This just fixes this warning.